.gitattributes

```diff
@@ -0,0 +1,4 @@
+package-lock.json linguist-generated=true -diff
+yarn.lock linguist-generated=true -diff
+bindgen/dist/**/*.js linguist-generated=true -diff
```

Comment:
We would want to ignore `.ts` files in this dir, too. Right? 

Maybe `.gitattributes` allows you to ignore the whole dir – that would be ideal